:ambulance: Hotfix  in readme (#10)
ps-thunderatz/sumo_simulation: README.md

- **mini_match.launch** - Arquivo que invoca a pista com dois sumôs em posição inicial de luta.

```bash
roslaunch sumo_simulation mini_match.launch
```

- **tiebreaker.launch** - Arquivo que invoca apenas os dois sumôs em posição de desempate.

```bash
roslaunch sumo_simulation tiebreaker.launch
```

- **gazebo.launch** - Arquivo que contém o dojô, que será usado durante os combates.

```bash
roslaunch sumo_simulation gazebo.launch
```

- **spawn.launch** - Arquivo que invoca apenas um dos robôs. Para especificar a cor do time é necessário adicionar ```team:=cor_do_time``` no final do comando.

```bash
roslaunch sumo_simulation spawn.launch team=:red
```

- **robots.launch** - Arquivo que invoca apenas os dois sumôs em posição para lutar.

```bash
roslaunch sumo_simulation robots.launch
```

- **calibrate_line.launch** - Arquivo para iniciar a calibração.

```bash
roslaunch sumo_simulation calibrate_line.launch
```

## :joystick: Controle

Para controlar os sumôs, inicie o [pacote de controle](https://github.com/ps-thunderatz/sumo_controller/) de acordo com o ensinado durante as aulas e no README dele!

## :pushpin: Tópicos

> Para controlar algumas partes da simulação, o ROS funciona se **inscrevendo** e **publicando** em tópicos. Para ter acesso aos tópicos em que a simulação está publicando, basta abrir um novo terminal e utilizar o seguinte comando:

Reviewer: ```suggestion
Para controlar algumas partes da simulação, o ROS funciona se **inscrevendo** e **publicando** em tópicos. Para listar os tópicos que a simulação está publicando, basta abrir um novo terminal e utilizar o seguinte comando:
```

```bash
rostopic list
```

### :arrow_left: Entrada

> Para controlar a velocidade do robô é necessário se inscrever nos tópicos do ROS de cada roda. A simulação recebe mensagens do tipo[std_msgs/Float64](http://docs.ros.org/en/noetic/api/std_msgs/html/msg/Float64.html). Os tópicos que controlam as rodas são:

Reviewer: ```suggestion
Para controlar a velocidade do robô é necessário publicar nos tópicos do ROS de cada roda. A simulação recebe mensagens do tipo[std_msgs/Float64](http://docs.ros.org/en/noetic/api/std_msgs/html/msg/Float64.html). Os tópicos que controlam as rodas são:
```

- **robot_left_controller/command**
- **robot_right_controller/command**

O controle do robô é feito pelo [sumo_controller](https://github.com/ps-thunderatz/sumo_controller). Nesse repositório, existem bibliotecas que auxiliarão a movimentação.

### :arrow_right: Saída

O pacote de simulação publica alguns tópicos que serão importantes no desenvolvimento de estratégias no [sumo_controller](https://github.com/ps-thunderatz/sumo_controller). Os tópicos publicados são:

> - **distance_sensor_[0..2]/reading** - Tópico que realiza a leitura dos dados dos sensores de distância.
> - **imu/** - Tópico que realiza a leitura dos dados da IMU. A partir dessa leitura é possível, no pacote de controle, receber a velocidade angular e a aceleração linear do robô.
> - **left_line_sensor/reading** - Tópico que realiza a leitura do sensor de linha no lado esquerdo do robô.
> - **right_line_sensor/reading** - Tópico que realiza a leitura do sensor de linha no lado direito do robô.

Reviewer: ```suggestion
- **distance_sensor_[0..2]/reading** - Tópico da leitura dos dados dos sensores de distância.
- **imu/** - Tópico da leitura dos dados da IMU. A partir dessa leitura é possível, no pacote de controle, receber a velocidade angular e a aceleração linear do robô.
- **left_line_sensor/reading** - Tópico da leitura do sensor de linha no lado esquerdo do robô.
- **right_line_sensor/reading** - Tópico da leitura do sensor de linha no lado direito do robô.
```

## :joystick: Controle

Para controlar os sumôs, inicie o [pacote de controle](https://github.com/ps-thunderatz/sumo_controller/) de acordo com o ensinado durante as aulas e no README dele!

## :pushpin: Tópicos

> Para controlar algumas partes da simulação, o ROS funciona se **inscrevendo** e **escrevendo** em tópicos. Para ter acesso aos tópicos que estão sendo publicados na simulação, basta abrir um novo terminal e utilizar o seguinte comando:

Reviewer 2: Acho legal usar os mesmos termos que em inglês né, pra não ter confusão.

```suggestion
Para controlar algumas partes da simulação, o ROS funciona se **inscrevendo** e **publicando** em tópicos. Para ter acesso aos tópicos em que a simulação está publicando, basta abrir um novo terminal e utilizar o seguinte comando:
```

> ```ros

Reviewer: ```suggestion
```bash
```

rostopic list

> ```

Reviewer: Acho que dá pra colocar os tópicos que nem [aqui](https://github.com/ThundeRatz/travesim/blob/develop/README.pt-br.md#-t%C3%B3picos-ros), porque o Gazebo publica várias coisas, nem todas são importantes pra gente, aí acho que pode confundir.

Reviewer 2: Eh que também faz parte da tarefa eles descobrirem né kkkkkk
